Marlin/Configuration_adv.h

```diff
@@ -1262,6 +1262,7 @@
 //#define CUSTOM_USER_MENUS
 #if ENABLED(CUSTOM_USER_MENUS)
   #define USER_SCRIPT_DONE "M117 User Script Done"
+  #define USER_SCRIPT_FEEDBACK    // Disable to turn off audible feedback when user script is executed
```

Comment:
I like that.  I have a predisposition towards short variable names, but I'm learning that in Marlin we like long, descriptive, names.  I made it `USER_SCRIPT_AUDIBLE_FEEDBACK` because `USER_SCRIPT_ENABLE_AUDIBLE_FEEDBACK` just seems to long, and `#if ENABLED(USER_SCRIPT_AUDIBLE_FEEDBACK` groks better than `#if ENABLED(USER_SCRIPT_ENABLE_AUDIBLE_FEEDBACK`, IMHO.